scripts/python/services/GithubService.py

```diff
@@ -4,24 +4,30 @@
 
 from github import Github
 from github.Issue import Issue
+from gql import Client
+from gql.transport.aiohttp import AIOHTTPTransport
 
 
 class GithubService:
     USER_ACCESS_REMOVED_ISSUE_TITLE: str = "User access removed, access is now via a team"
 
     def __init__(self, org_token: str, organisation_name: str) -> None:
-        self.client: Github = Github(org_token)
+        self.github_client_core_api: Github = Github(org_token)
+        self.github_client_gql_api: Client = Client(transport=AIOHTTPTransport(
+            url="https://api.github.com/graphql",
+            headers={"Authorization": f"Bearer {org_token}"},
+        ), fetch_schema_from_transport=False)
```

Comment:
Removed exception handling that was present before - if an exception is thrown here, we'll throw it to the REPL now to stop the execution of the script